rust/stracciatella/src/schemas/yaml/vehicles.schema.yaml

```diff
@@ -0,0 +1,51 @@
+$schema: http://json-schema.org/draft-04/schema#
+type: array
+title: Vehicles
+items:
+  type: object
+  title: Vehicle
+  description: |
+    Definitions of player-controllable transport vehicle.
+
+    The ordering in this list is important, and the game has hard-coded references to the
+    vehicles. For example, the game expects the HELICOPTER to be the 6th vehicle.
+
+    In base vanilla game, only the Hummer, Ice-cream truck and Helicopter are available.
+  properties:
+    enterSound:
+      title: Enter sound
+      description: Sound file to play when a soldier enters the vehicle.
+      $ref: types/resource-path.schema.yaml
+    moveSound:
+      title: Move sound
+      description: Sound file to play when the vehicle moves.
+      $ref: types/resource-path.schema.yaml
+    profileID:
+      title: Profile ID
```

Comment:
Should we use a different type for merc profile ID? It should be always be `0 < n <= 200`.

Reply:
As one of my goals is to replace all occurences of `profileID` with a string id pointing to `mercs-profile-info.json` I will add it to `mercs-profile-info.json` and leave it at that.